util/srvpk.go

```diff
@@ -0,0 +1,28 @@
+/*srvpk.go*/
+package main
+import (
+	"net/http"
+	"log"
+	"os"
+	"github.com/bitfield/script"
+	"github.com/gorilla/mux"
+)
+
+
+func ServePK(w http.ResponseWriter, r *http.Request) {
+	pk, err := script.Exec(`skywire-cli visor pk`).Bytes()
+	if err != nil {
+	log.Printf("error occured")
+	os.Exit(1)
+}
+	w.Write([]byte(pk))
+}
+
+func main() {
+    r := mux.NewRouter()
+    // Routes consist of a path and a handler function.
```

Comment:
There is no need to use gorilla here. Also the comments are superfluous imo.

Reply:
i know its overkill. will see about adding a stripped down version of this this to skywire-cli tomorrow

Reply:
solved by this PR:
https://github.com/skycoin/skywire/pull/1270

removed from this repo / PR